spec/ruby-units/unit_spec.rb

```diff
@@ -1353,7 +1353,7 @@
 
   context '#divmod' do
     specify { Unit("5 mm").divmod(Unit("2 mm")).should == [2, 1] }
-    specify { Unit("1 km").divmod(Unit("2 m")).should == [500, 0] }
+    specify { Unit("1 km").divmod(Unit("2 m")).should == [Unit('500 km/m'), Unit('0 km/m')] }
```

Comment:
This is wrong.  This test says that if you divide '1000 m' by '2 m', you should get [500_000, 0] instead of [500, 0].

Note that `Unit('500 km/m').to_base #=> 500000`

Also note that `Unit('50%').to_base #=> 1/2`